src/MobileMapping/Sensor.js

```diff
@@ -87,6 +87,7 @@ Sensor.prototype.getMatOrientationCapteur = function getMatOrientationCapteur()
         case 2:
             return ori2;
         case 3:
+        default:
```

Comment:
Definining an arbitrary default value can't hurt, can it? (the alternative would be to raise an error I guess)